src/maptm/src/maptm_ovsdb.c

```diff
@@ -148,21 +148,20 @@ bool maptm_persistent(void)
     }
 
```

Comment:
About the use of osp_ps_get in "size = osp_ps_get(ps, "MAPT_SUPPORT", NULL, 0);":
This API call is used to check the existence of persistent storage. Can Plume provide more details to help us fix the flawed use of this API?